corehq/apps/couch_sql_migration/casediff.py

```diff
@@ -617,6 +617,24 @@ def clean_break(self, value):
         self.queue.clean_break = value
 
 
+class NoCaseDiff(object):
```

Comment:
No action requested, just a Python 3 fun fact: [Python 3 doesn't have old-style/classic classes](https://www.python.org/doc/newstyle/), so (if code will not be run on Python 2) [we can drop "`(object)`"](https://docs.python.org/3/tutorial/classes.html#class-definition-syntax).

Reply:
d'oh! I know this, but my brain reverted to old-style for some reason.

Reply:
Muscle memory :grin: :keyboard: :muscle: